### PR TITLE
Added details of last emotly per user.

### PR DESCRIPTION
**Pull request by tizianasellitto**

### CODE REVIEW

File: emotly/constants.py [hunk 1]
```diff
@@ -39,6 +39,7 @@
 USER_DOES_NOT_EXIST = 'Authentication error.'
 
 EMOTLY_DOES_NOT_EXIST = 'Emotly error.'
+NICKNAME_DOES_NOT_EXIST = 'Nickname does not exist.'
```
Comment (michelangelo) [inline emotly/constants.py]: `USER_DOES_NOT_EXIST = 'User does not exist.'`

File: emotly/controllers/emotly_controller.py [hunk 2]
```diff
@@ -96,6 +96,23 @@ def get_emotly(emotly_id, **kwargs):
     return response_handler(200, emotly.serialize(), 'emotly')
 
 
+# Retrieve the last emotly of a specific user.
+@emotly_controller.route(CONSTANTS.REST_API_PREFIX + '/user/<nickname>',
+                         methods=['GET'])
+@require_token
```
Comment (michelangelo) [inline emotly/controllers/emotly_controller.py]: Do we want to restrict this to logged users only? Your call, though.

File: emotly/tests/rest_api_tests.py [hunk 3]
```diff
@@ -58,10 +58,22 @@ def test_moods_type(self):
             self.assertEqual(type(mood['id']), int)
             self.assertEqual(type(mood['value']), str)
 
+    # Next tests user unauthorized access to api.
```
Comment (michelangelo) [inline emotly/tests/rest_api_tests.py]: ???

File: emotly/tests/rest_api_tests.py [hunk 4]
```diff
@@ -315,3 +381,53 @@ def test_not_confirmed_user_get_emotly(self):
...
+                 salt="salt")
+        u.save()
+        headers = {'content-type': 'application/json',
+                   'X-Emotly-Auth-Token': generate_jwt_token(u)}
+
+        rv = self.app.get(CONSTANTS.REST_API_PREFIX + '/user/' + u.nickname,
+                          headers=headers)
+        self.assertEqual(rv.status_code, 403)
+
+    def test_get_user_details_last_emotly_nickname_non_existent(self):
```
Comment (michelangelo) [inline emotly/tests/rest_api_tests.py]: Spelling.

**Comment by michelangelo** [comment]
Hi @tizianasellitto, would you please rebase this? We've recently landed https-by-default and this doesn't merge cleanly.:)

**Comment by tizianasellitto** [comment]
Done

**Comment by michelangelo** [comment]
Great, thanks! Nice work, LGTM.

**Pull request merged**